openapi/saps4hana.itcm.promotion/Dependencies.toml

```diff
@@ -0,0 +1,9 @@
+[[dependency]]
+org = "ballerina"
+name = "http"
+version = "2.0.0"
+
+[[dependency]]
```

Comment:
``` 
org = "ballerina"
name = "url"
version = "2.0.0" 
```

This dependency is not used anywhere. Shall we remove it?

Reply:
Yes. This dependency is removed